regression/cbmc-incr-smt2/nondeterministic-int-assert/nondet_symbol.c

```diff
@@ -0,0 +1,11 @@
+extern int john();
```

Comment:
:no_entry_sign: This `.c` file for a test appears to be missing the corresponding `.desc` file. I assume you just forgot to commit it?